commercial/test/model/hosted/HostedTrailsTest.scala

```diff
@@ -53,6 +53,7 @@ class HostedTrailsTest extends FlatSpec with Matchers {
         }
       )
     )
+    val charts = None
```

Comment:
Out of interest, what does this do? Is it related to the version bump?

Reply:
Yep, a new atom type `Chart` has been created and is supported in 12.10. This change just gets things compiling.